shim/shim.py

```diff
@@ -440,6 +440,11 @@ def _add_configuration_arguments(parser: argparse.ArgumentParser) -> None:
             "Use --emit-all-via-cast-features to allow everything."
         ),
     )
+    configuration_arguments.add_argument(
+        "--propagate-via-arg",
+        action="store_true",
+        help="Enable taint propagation via object type arguments.",
+    )
```

Comment:
I think this deserves a better name and better documentation.
The important fact is that this infers propagations between  arguments >= 1.
What about calling it `--propagate-between-arguments`?
Could you also add something about it in the documentation? We have a section about that false negative:
https://www.internalfb.com/intern/staticdocs/mariana-trench/docs/fb/known-false-negatives/#propagation-across-arguments

Reply:
I changed the property name and updated the help text. However, I do not think that false negative section is available in the repository for us externals. Perhaps I could add a section to `documentation/website/documentation/configuration.md`?

Reply:
Could you rebase on top of 64c08cd86544cbbdd23fddf824795b306c330c82 and add something to the documentation now?